src/component/rating/Rating.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import 'bulma/css/bulma.css';
+import './Rating.css'
+
+const Rating = props =>{
+    const ratingNumber = Math.round(props.children)
+    let stars = ''
+    switch (ratingNumber){
+        case 0:
+            stars = '\u2606 \u2606 \u2606 \u2606 \u2606'
+            break;
+        case 1:
+            stars = '\u2605 \u2606 \u2606 \u2606 \u2606'
+            break;
+        case 2:
+            stars = '\u2605 \u2605 \u2606 \u2606 \u2606'
+            break;
+        case 3:
+            stars = '\u2605 \u2605 \u2605 \u2606 \u2606'
+            break
+        case 4:
+            stars = '\u2605 \u2605 \u2605 \u2605 \u2606'
+            break;
+        case 5:
+            stars = '\u2605 \u2605 \u2605 \u2605 \u2605'
+            break;
+        default:
+            stars = '\u2606 \u2606 \u2606 \u2606 \u2606'
+            break;
+    }
+    
+    return(
+        <div className="card">
+            <div className="card-content">
+                <div className="columns is-mobile">
+                    <div className="column is-12">
+                        <div className="media">
+                            {stars}
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    )
+}
+
```

Comment:
Damn bud!!! It looks great but you just had to copy the star from the lab text like this : ☆☆☆☆☆ xD

```
function Rating(props) {
  
 let num = Math.round(props.children);

  let star = "";
  switch (num) {
    case 0:
      star = "☆☆☆☆☆";
      break;
    case 1:
      star = "★☆☆☆☆";
      break;
    case 2:
      star = "★★☆☆☆";
      break;
    case 3:
      star = "★★★☆☆";
      break;
    case 4:
      star = "★★★★☆";
      break;
    case 5:
      star = "★★★★★";
      break;
    default:
      break;
  }

  return <div>{star}</div>;
}
```